Source/IGListAdapter.m

```diff
@@ -720,14 +720,35 @@ - (__kindof UICollectionViewCell *)dequeueReusableCellOfClass:(Class)cellClass
     return [collectionView dequeueReusableCellWithReuseIdentifier:identifier forIndexPath:indexPath];
 }
 
+- (UICollectionViewCell *)dequeueReusableCellFromNibName:(NSString *)nibName
+                                                  bundle:(NSBundle *)bundle
+                                               cellClass:(Class)cellClass
+                                    forSectionController:(IGListSectionController<IGListSectionType> *)sectionController
+                                                 atIndex:(NSInteger)index {
+    IGAssertMainThread();
+    IGParameterAssert(nibName != nil);
+    IGParameterAssert(sectionController != nil);
+    IGParameterAssert(cellClass != nil);
+    UICollectionView *collectionView = self.collectionView;
+    IGAssert(collectionView != nil, @"Reloading adapter without a collection view.");
+    NSString *identifier = IGListReusableViewIdentifier(cellClass, nibName, nil);
+    NSIndexPath *indexPath = [self indexPathForSectionController:sectionController index:index];
+    if (![self.registeredCellClasses containsObject:cellClass]) {
+        [self.registeredCellClasses addObject:cellClass];
+        UINib *nib = [UINib nibWithNibName:nibName bundle:bundle];
```

Comment:
Noting to myself that we don't need an assert to check if the nib exists b/c according to [the docs](https://developer.apple.com/reference/uikit/uinib/1614138-init):

> An exception is thrown if there were errors during initialization or the nib file could not be located.